jobbergate-cli/jobbergate_cli/subapps/job_submissions/tools.py

```diff
@@ -56,23 +98,44 @@ def create_job_submission(
         ),
     )
 
+    Abort.require_condition(
+        not (settings.SBATCH_PATH is not None and execution_parameters_file is not None),
```

Comment:
You can De Morgan (!(A and B) == !A or !B) this expression to:
`settings.SBATCH_PATH is None or execution_parameters_file is None`

Reply:
Solved in [e8c887a](https://github.com/omnivector-solutions/jobbergate/pull/427/commits/e8c887a998c9a963f25b84b7561be332c4461ce9)